### ISSUE TITLE
docker image

### ISSUE THREAD

**Issue by abdusalamov** [Bashka/taskbot#3]
Что на счет докер-версии? :) Штуку попробовать хочу, но поднимать php — не очень. 

Было бы полезно собрать готовый образ и загрузить на докерхаб.

**Comment by vintersorg** [comment]
согласен. с докером было бы проще понять насколько функционал подходит.
